src/assets/locales/en.js

```diff
@@ -167,6 +167,20 @@ export default {
           duplicate: "Duplicate",
           analyse: "Analyse",
           share: "Share",
+          delete: "Delete",
+        },
+        dialog: {
+          delete: {
+            title: "Are you sure you want to delete this plio?",
+            description:
+              "This will permananently delete this plio and all the data associated with it, including all the responses. Once deleted, the data cannot be covered.",
```

Comment:
I don't think the bold is needed. For all delete stuff as well, we didn't keep anything bold. The dialog box itself is enough to prevent the user from taking erroneous action I think.